content/docs/hooks-reference.md

```diff
@@ -508,3 +514,59 @@ For example a custom Hook that returned a `Date` value could avoid calling the `
 ```js
 useDebugValue(date, date => date.toDateString());
 ```
+
+### `useDeferredValue` {#usedeferredvalue}
+
+```js
+const [deferredValue] = useDeferredValue(value);
+```
+
+TODO: description
+
+### `useTransition` {#usetransition}
+
+```js
+const [isPending, startTransition] = useTransition();
+```
+
+TODO: description
+
+## Library Hooks {#library-hooks}
+
+The following Hooks are provided for library authors to integrate libraries deeply into the React model, and are not typically used in application code.
+
+### `useId` {#useid}
+
+```js
+const id = useId(value);
+```
+
+TODO: description
+
+> Note:
+> 
+> TODO: identifierPrefix
+
+### `useInsertionEffect` {#useinsertioneffect}
```

Comment:
Maybe we put this last since it's less common than useSES.

Reply:
Yeah, and @acdlite made a good point that `useId` isn't strictly 'library' so I moved that up as well.